### PR TITLE
Profile page

### PR DESCRIPTION
**Pull request by rahulyadav139**
Dmo: https://golden-griffin-8345a7.netlify.app/

### CODE REVIEW

File: src/pages/homepage/Homepage.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
+.main-homepage {
+  /* padding: 1rem 8rem; */
+  max-width: 1000px;
```
Comment (mehulsatardekar) [inline src/pages/homepage/Homepage.css:3]: use rems here

File: src/components/suggestion-card/SuggestionCard.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import './SuggestionCard.css';
+
+const SuggestionCard = props => {
+  return (
+    <div className=" suggestion-card ">
+      <div className="avatar small">
+        <img
+          src="https://i.picsum.photos/id/933/536/354.jpg?hmac=8lVRoNcysARFInMz443q-mc0wbgwHbJgFe5ChEo-YaQ"
+          alt=""
```
Comment (mehulsatardekar) [inline src/components/suggestion-card/SuggestionCard.js:9]: use alt text for image

File: src/components/social-media-card/SocialMediaCard.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+const SocialMediaCard = props => {
+  return (
+    <div class="card shadow social">
+      <div class="flex space-between align-center">
+        <div class="flex gap align-center">
+          <div class="avatar small">
+            <img
+              src="https://i.picsum.photos/id/933/536/354.jpg?hmac=8lVRoNcysARFInMz443q-mc0wbgwHbJgFe5ChEo-YaQ"
+              alt=""
```
Comment (mehulsatardekar) [inline src/components/social-media-card/SocialMediaCard.js:11]: use alt text text for image

**Review by mehulsatardekar — APPROVED** [review]

File: src/components/post-image-card/PostImageCard.css [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+  cursor: pointer;
+}
+.post-details > div {
+  display: flex;
+  gap: 2.5rem;
+}
+
+/* 
+.post-details-overlay {
+  width: 100%;
```
Comment (mehulsatardekar) [inline src/components/post-image-card/PostImageCard.css:33]: remove commented code

File: src/components/post-image-card/PostImageCard.js [hunk 5]
```diff
@@ -0,0 +1,33 @@
+import './PostImageCard.css';
+
+const PostImageCard = props => {
+  return (
+    <div className="post-image-card">
+      <img
+        src="https://i.picsum.photos/id/933/536/354.jpg?hmac=8lVRoNcysARFInMz443q-mc0wbgwHbJgFe5ChEo-YaQ"
+        alt=""
```
Comment (mehulsatardekar) [inline src/components/post-image-card/PostImageCard.js:8]: add alt text for an image

**Review by MonikaJhingan — APPROVED** [review]
The code structure is well organised, needs to be merged.

File: src/pages/account/Account.css [hunk 6]
```diff
@@ -0,0 +1,31 @@
+.main-account {
+  max-width: 800px;
```
Comment (mehulsatardekar) [inline src/pages/account/Account.css:2]: use rems for width sizing

File: src/pages/profile/Profile.css [hunk 7]
```diff
@@ -0,0 +1,48 @@
+.main-profile-page {
+  max-width: 1000px;
```
Comment (mehulsatardekar) [inline src/pages/profile/Profile.css:2]: use rems here

**Review by mehulsatardekar — APPROVED** [review]

**Pull request merged**